plugins/bufbuild/connect-kotlin/v0.1.0/buf.plugin.yaml

```
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/bufbuild/connect-kotlin
+plugin_version: v0.1.0
+source_url: https://github.com/bufbuild/connect-kotlin
+description: Idiomatic gRPC & Connect RPCs for Kotlin.
+deps:
+  - plugin: buf.build/protocolbuffers/kotlin:v21.12
```

```suggestion
  - plugin: buf.build/protocolbuffers/kotlin:v22.0
```

We should bump this to the latest version.